RevenueCatUI/Data/PaywallViewConfiguration.swift

```diff
@@ -20,6 +21,7 @@ struct PaywallViewConfiguration {
     var displayCloseButton: Bool
     var introEligibility: TrialOrIntroEligibilityChecker?
     var purchaseHandler: PurchaseHandler?
+    var uiUserInterfaceStyle: UIUserInterfaceStyle?
```

Comment:
This can be simpler:
```suggestion
    var userInterfaceStyle: UIUserInterfaceStyle?
```